gulpfile.babel.js

```diff
@@ -247,6 +249,19 @@ gulp.task('json', function() {
 });
 
 
+const accessAsync = promisify(fs.access);
```

Comment:
What you have here is fine, but one option you might consider is the [`fs-extras`](https://github.com/jprichardson/node-fs-extra) package, which is really popular, is more flexible than the native `fs` package, and comes with all it's methods already promisified.

Reply:
Done